src/Components/Dialogs/dialogs.jsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
 
 import style from './dialogs.module.css';
+import I18Y from '../../core/constants';
```

Comment:
Consider fixing the grouping of these imports.

Reply:
I changed group